package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@folio/plugin-find-authority",
+  "version": "1.0.0",
+  "description": "Stripes plugin to find authorities",
+  "repository": "folio-org/ui-plugin-find-authority",
+  "publishConfig": {
+    "registry": "https://repository.folio.org/repository/npm-folio/"
+  },
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=14.0.0"
+  },
+  "stripes": {
+    "actsAs": [
+      "plugin"
+    ],
+    "pluginType": "find-authority",
+    "displayName": "ui-plugin-find-authority.meta.title",
+    "queryResource": "query",
+    "permissionSets": [
+      {
+        "permissionName": "module.ui-plugin-find-authority.enabled",
+        "displayName": "UI: find authorities plugin is enabled"
+      }
+    ]
+  },
+  "scripts": {
+    "lint": "eslint .",
+    "test": "jest",
+    "formatjs-compile": "formatjs compile-folder --ast --format simple ./translations/ui-plugin-find-authority ./translations/ui-plugin-find-authority/compiled"
+  },
+  "okapiInterfaces": {
+    "eholdings": "2.0"
```

Comment:
We don't need `eholdings` interface, but need 
```
"search": "0.6",
"browse": "0.3",
"source-storage-records": "3.0",
"records-editor.records": "3.1"
```

Reply:
Thanks, fixed